symplyphysics/laws/kinematic/classical_addition_of_velocities.py

```diff
@@ -0,0 +1,54 @@
+from sympy import Eq
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## The law of classical addition of velocities, usually attributed to Galileo and called the
+## Galilean law of velocity addition, states that the velocity of a body in an inertial reference
+## frame A can be found as a sum of its velocity in another inertial reference frame B and the
+## velocity of frame B relative to frame A.
+
+# Law: u = v + u'
+## u - velocity of body relative to frame A
+## u' - velocity of body relative to frame B
+## v - velocity of frame B relative to frame A
+
+# Conditions
+## - Space and time are absolute
+## - Applies to inertial frames of reference
+
+speed_relative_to_first_frame = Symbol("velocity_relative_to_first_frame", units.velocity)
+speed_relative_to_second_frame = Symbol("velocity_relative_to_second_frame", units.velocity)
+relative_speed_of_second_frame_relative_to_first = Symbol(
```

Comment:
`relative` twice.
What if we use more consistent naming?
Eg, `body_in_first_frame_velocity`, `body_in_second_frame_velocity`, `second_frame_in_first_frame_velocity`?

Reply:
I agree, this is a better choice